snuba/web/query.py

```diff
@@ -115,6 +120,7 @@ def _run_query_pipeline(
     query_runner = partial(
         _format_storage_query_and_run,
         dataset,
+        reader,
```

Comment:
So, reader is passed to runner from the execution strategy rather than `_run_query_pipeline`? That sounds good to me and lets us remove the `get_cluster()` which only existed for that reason.

Re: 1 and 2, is there a reason why runner should be changed to take the cluster as well instead of just the reader like before though?

Reply:
Runner is not taking the cluster not the reader now. It is just the partially applied function that gets the reader. I am talking about this https://github.com/getsentry/snuba/blob/master/snuba/datasets/plans/query_plan.py#L12

And I would pass cluster instead of reader because it is a more abstract concept, and doing that does not add complexity.

Reply:
Sorry, the last part was supposed to read `_format_storage_query_and_run` is taking the reader and passing it through to `raw_query`, not runner.